example.env

```diff
@@ -61,12 +61,9 @@ PDM_POSTGRES_USER=test
 PDM_POSTGRES_PASSWORD=test
 PDM_POSTGRES_PORT=5432
 
-PDM_BACKEND_CORS_ORIGINS=["*"]
+CORS_ORIGINS=["*"]
 PDM_SERVICE_PORT=8003
-SERVICE_NAME=pdm_backend
-
-PDM_ACCESS_TOKEN_EXPIRATION_TIME=240
-PDM_JWT_ALGORITHM=HS256 #ex. HS256
+SERVICE_NAME=pdm_backend # This should be changed to a standardised service name like pestdismgmt or pdm, etc.
 
 PDM_USING_GATEKEEPER=True
-PDM_GATEKEEPER_BASE_URL=http://gatekeeper:8001
+PDM_GATEKEEPER_BASE_URL=http://gatekeeper:8001 # This should be removed. Instead use INTERNAL_GK_URL declared above
```

Comment:
This should be removed. Instead use INTERNAL_GK_URL declared above

Reply:
OK I see the mismatch, but we need to agree in a common format if we do not want to have duplicated the same var

Reply:
I propose to leave it like this for now, but open an issue on PDM for a small change in the future